templates/account/payments.php

```diff
@@ -14,15 +14,11 @@
 	 * @var array $VARS
 	 * @var Freemius $fs
 	 */
-	$fs = freemius( $VARS['id'] );
+    $fs       = freemius( $VARS['id'] );
+    $payments = freemius( $VARS['payments'] );
```

Comment:
@msladu for IDE autocompletion and improved documentation, unlike previously using `_fetch_payments()` the code could resolve the type of the variable because of the `@return` param of the method, now we need to explicitly specify it. We do it by adding a little comment above the relevant variable using the `@var` directive as following:
```
/**
 * @var FS_Payment[] $payments
 */
```

Reply:
@vovafeldman Ok.